### PR TITLE
Add sklearn.ensemble.HistGradientBoostingRegressor

### LINKED ISSUE #127: Add sklearn.ensemble.HistGradientBoostingRegressor
This should be easy since we already have `HistGradientBoostingClassifier` there.

### PR DESCRIPTION
**Pull request by TuanNguyen27**
Solves #127

### CODE REVIEW

**Comment by TuanNguyen27** [comment]
@interesaaat i got a little stuck at converting `tree_infos` into the right format. Wondering if you can take a look. Thanks!

**Comment by interesaaat** [comment]
Hi @TuanNguyen27, I will try to take a look today!

**Comment by TuanNguyen27** [comment]
@interesaaat awesome work! Looks like you solved the mismatch in prediction values. How did you figure out that there's no learning rate for `HistGradientBoostingRegressor` ?

**Comment by interesaaat** [comment]
We had the same problem for the classification operator :)

**Review by ksaur — APPROVED** [review]
LGTM

**Comment by TuanNguyen27** [comment]
thx @ksaur ! Would you mind merging it as well ? I don't think I have merge power on this repo
